### PR TITLE
Fix js mistakes and change postgres version and  psycopg2 version

### PR DESCRIPTION
**Pull request by AndreyLykhoman**
Description:  This commit has:
 - Change version of psycopg2, and write the version of Postgres in the docker-compose file.
 - Fix grading_policy_form URL. 
 - Fix js script for reloading a page when a form already sent.

Main Task:   https://youtrack.raccoongang.com/issue/ALOSI-109
Task for review: https://youtrack.raccoongang.com/issue/ALOSI-112

### CODE REVIEW

File: bridge_adaptivity/module/static/module/js/module.js [hunk 1]
```diff
@@ -360,9 +360,12 @@
         const modalSuccessListener = (url) => {
             return result => {
                 let modal = $('#modal-wrapper');
-                if (result.status !== undefined && result.status === 'ok' && result.collection_slug !== undefined) {
-                    var new_url = new URL(window.location.href);
+                var new_url = new URL(window.location.href);
+                 // FixMe: This check  is needed to remove in another place
```
Comment (idegtiarov) [inline bridge_adaptivity/module/static/module/js/module.js]: Please rewrite the comment id doesn't clear enough.

File: bridge_adaptivity/module/static/module/js/module.js [hunk 2]
```diff
@@ -360,9 +360,12 @@
         const modalSuccessListener = (url) => {
             return result => {
                 let modal = $('#modal-wrapper');
-                if (result.status !== undefined && result.status === 'ok' && result.collection_slug !== undefined) {
-                    var new_url = new URL(window.location.href);
+                var new_url = new URL(window.location.href);
```
Comment (idegtiarov) [inline bridge_adaptivity/module/static/module/js/module.js]: `let` definition for the `new_url` variable will be faire enough.

File: bridge_adaptivity/module/static/module/js/module.js [hunk 3]
```diff
@@ -360,9 +360,12 @@
...
+                 // FixMe: This check  is needed to remove in another place
+                if (result.collection_slug !== undefined) {
                     new_url.searchParams.append('collection_slug', result.collection_slug);
+                }
+                if (result.status !== undefined && result.status === 'ok') {
```
Comment (idegtiarov) [inline bridge_adaptivity/module/static/module/js/module.js]: Why do we need two statements it seems that second covers both cases.

File: bridge_adaptivity/module/static/module/js/module.js (hunk 1 above)
Comment (AndreyLykhoman) [reply bridge_adaptivity/module/static/module/js/module.js]: Done

File: bridge_adaptivity/module/static/module/js/module.js (hunk 2 above)
Comment (AndreyLykhoman) [reply bridge_adaptivity/module/static/module/js/module.js]: Done

File: bridge_adaptivity/module/static/module/js/module.js (hunk 3 above)
Comment (AndreyLykhoman) [reply bridge_adaptivity/module/static/module/js/module.js]: The first statement is used when the collection id changed. And it's used by only one flow: When we use CollectionUpdate view
The second state is used by all flows to create or update something. When a form saved correctly, we could reload the page.

Comment (idegtiarov) [reply bridge_adaptivity/module/static/module/js/module.js]: I meant `result.status !== undefine` is totally the same as we check `result.status === 'ok'`. So why do we need the first check?

Comment (AndreyLykhoman) [reply bridge_adaptivity/module/static/module/js/module.js]: You are right! We don't need the first check. I fixed this.

**Pull request merged**